pkg/cmd/pulumi/console.go

```diff
@@ -64,6 +64,11 @@ func newConsoleCmd() *cobra.Command {
 					if err != nil {
 						return err
 					}
+					if stack == nil {
+						fmt.Println("No stack is currently selected. " +
+							"Please run `pulumi stack select` to select a stack.")
```

Comment:
```suggestion
							"Run `pulumi stack select` to select a stack.")
```

we should try and avoid using please unless we are asking the person to do something inconvenient, or pulumi is at fault.

Reply:
Thank you for the tip!